tests/common/tracker.rs

```diff
@@ -0,0 +1,34 @@
+use std::sync::Arc;
+
+use torrust_tracker::tracker::statistics::Keeper;
+use torrust_tracker::tracker::Tracker;
+use torrust_tracker::{ephemeral_instance_keys, logging, static_time};
+
+pub fn tracker_configuration() -> Arc<torrust_tracker_configuration::Configuration> {
+    Arc::new(torrust_tracker_test_helpers::configuration::ephemeral())
+}
+
+// TODO: Move to test-helpers crate once `Tracker` is isolated.
+pub fn tracker_instance(configuration: &Arc<torrust_tracker_configuration::Configuration>) -> Arc<Tracker> {
```

Comment:
hi @WarmBeer `tracker_isntance` makes me think that's a singleton and not a factory. Maybe `tracker_factory` or `new_tracker_instance`. I would choose the first one because it's not a constructor; it's a dependency "resolver". In fact, I think this is the app container and bootstrapping at the same time.